easybuild/easyconfigs/p/pythran/pythran-0.9.4.post1-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'pythran'
+version = '0.9.4.post1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pythran.readthedocs.io'
+description = """ Pythran is an ahead of time compiler for a subset of the Python language,
+with a focus on scientific computing. It takes a Python module annotated with a few interface description
+and turns it into a native Python module with the same interface, but (hopefully) faster. """
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('networkx', '2.4', versionsuffix),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('ply', '3.11', {
+        'checksums': ['00c7c1aaa88358b9c765b6d3000c6eec0ba42abca5351b095321aef446081da3'],
+    }),
+    ('gast', '0.3.2', {
+        'checksums': ['5c7617f1f6c8b8b426819642b16b9016727ddaecd16af9a07753e537eba8a3a5'],
+    }),
+    ('beniget', '0.2.0', {
+        'checksums': ['4495fb8ef0e99f9291c751856e3cc0695115e7936acc54717a3f620ab331b247'],
+    }),
+    (name, version, {
+        'checksums': ['3842fefbb2374a28f8bb3e1ca9403513299438deb07608a5d76b1ae7b4fd24c0'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/'],
+}
+
+modloadmsg = """
+To use the OpenBLAS library, run:
+pythran --config=compiler.blas=openblas <python_script.py>
```

Comment:
@smoors Maybe it's better to define an alias for the `pythran` command for this in the module, so people can just use `pythran <python_script.py>`?

Reply:
@boegel good idea, see #9594